source/bin/tvm.rs

```diff
@@ -114,14 +115,24 @@ pub fn process_options(arguments: Vec<String>) {
                     },
 
                     _ => {
-                        println!("Invalid option “{}”.\n", argument);
+                        println!("Invalid option \"{}\".\n", argument);
                         println!("{}", usage());
                         exit(ExitCode::InvalidOption);
                     }
                 },
 
-            Some(_) =>
-                input = argument,
+            Some(_) => {
+                if input == None
+                {
+                    input = Some(argument);
+                }
+                else
+                {
+                    println!("Multiple input files\n");
```

Comment:
I was just saying to add the new filename we cannot handle. Having real data in the error message is always appreciated.